src/brevitas/nn/quant_scale_bias.py

```diff
@@ -31,9 +31,10 @@ def __init__(self, num_features: int, bias: bool, runtime_shape=(1, -1, 1, 1)):
         self.runtime_shape = runtime_shape
 
     def forward(self, input):
-        out = input * self.weight.view(self.runtime_shape)
+        dtype = input.dtype
+        out = input * self.weight.view(self.runtime_shape).to(dtype=dtype)
```

Comment:
Is there any way we can avoid casting on on the fly?
Also we cannot do this with `to.('cuda')` because that is definitely costly